cypress/e2e/registration.cy.js

```diff
@@ -1,11 +1,75 @@
 /// <reference types='cypress' />
 
-describe('Student Registration page', () => {
+describe( 'Student Registration page', () => {
+  
+  let user;
+
   before(() => {
-    
+    cy.visit('/')
+    cy.task('generateUser').then(generatedUser => {
+      user = generatedUser;
+    });
   });
 
-  it('', () => {
+  it('This form should allow to register a new user', () => {
     
+    cy.viewport(750, 600);
```

Comment:
Move cy.viewport(750, 600) to the 'before' hook to set the viewport only once before all test cases.